examples/demo_workflow/requirements.txt

```
@@ -0,0 +1 @@
+dapr-ext-workflow==1.5.0
```

```suggestion
dapr-ext-workflow-dev==0.0.1-rc.1
```

Where did you come up with this version? :D That will not be the version number we are using.

What version would you like to launch the SDK with? I would assume 0.1.0?

In the master branch we always import our development packages -- so use the `-dev` one. In the release branches we then remove the `-dev` suffix to refer to the stable packages. The `-dev` one is the builds from master that we constantly update.

---

Yeah, you are correct. I don't know how I wrote this number here :)
Will fix it.

---

Updated to `0.0.1rc1` as per version.py of dapr-ext-workflow